go/store/prolly/tuple_range.go

```diff
@@ -87,7 +90,7 @@ func (r Range) aboveStart(t val.Tuple) bool {
 			return false
 		}
 
-		if r.Fields[i].Exact && cmp == 0 {
+		if r.Fields[i].Unique && cmp == 0 {
```

Comment:
This doesn't seem correct to me, semantically -- it has to do with matching semantics, not unique semantics